packages/vote/fpc/fpc_test.go

```diff
@@ -35,6 +35,27 @@ func TestVoteContext_IsFinalized(t *testing.T) {
 	}
 }
 
+func TestVoteContext_HadFixedRound(t *testing.T) {
+	type testInput struct {
+		voteCtx               vote.Context
+		coolOffPeriod         int
+		finalizationThreshold int
```

Comment:
would it work if the create a moc context and set the parameters as we need them in the context?

Reply:
Now it is similar to how we provide it normally in the code. Those parameters are in FPC instanse paras. Do you mean changing the context struct definition?

Reply:
i was considering filling the context outside of the test with many of the fields empty. but as discussed your solution works good so lets keep it your way